src/main/resources/test_files/runtime/QuoteEscaping.jq

```diff
@@ -0,0 +1,2 @@
+(:JIQS: ShouldRun; Output="{ "foo\"foo\nbar" : "foo\"foo\nbar" }" :)
+json-file("./src/main/resources/quotes.json")
```

Comment:
The test file is actually located under ./src/main/resources/**queries**/quotes.json
I'm surprised that the testing suite works with either of these address inputs. The shell application only accepts the latter as a valid parameter. Throws an input path does not exist error otherwise

Reply:
Very good catch. The test wasn't executed because the extension must be .iq. I have fixed this.